packages/client/src/object/formatting/applyPropertyFormatter.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ObjectMetadata, PropertyValueFormattingRule } from "@osdk/api";
+import type { SimpleOsdkProperties } from "../SimpleOsdkProperties.js";
+import { formatBoolean } from "./formatBoolean.js";
+
+export interface FormatPropertyOptions {
```

Comment:
Github wasn't smart enough to understand a lot of this just moved from `applyPropertyFormatters`